mbed/mbed.py

```diff
@@ -3356,9 +3356,10 @@ def main():
         very_verbose = pargs.very_verbose
         verbose = very_verbose or pargs.verbose
         pathtype = Repo.pathtype(cwd_root)
-        action('Working path \"%s\" (%s)' % (cwd_root, pathtype))
-        if pathtype != "program":
-            action('Program path \"%s\"' % Program(cwd_root).path)
+        if not sys.argv[1].lower() in ["config"]: # `mbed config` should be parse-friendly
+            action('Working path \"%s\" (%s)' % (cwd_root, pathtype))
+            if pathtype == "library":
```

Comment:
Why did this line need to change from `==` to `!=` ?

Reply:
Because when you use mbed CLI on a directory that cannot be associated with a project, getting "Program path" message is very confusing - it may not be program at all. Also in the case of folder, the two lines report the same folder.

Reply:
Kk. Was primarily wondering if some third pathtype was present elsewhere, but that makes sense.